src/Compatibility/Core/src/iOS/Renderers/TabletFlyoutPageRenderer.cs

```diff
@@ -391,7 +392,8 @@ public override void WillRotate(UIInterfaceOrientation toInterfaceOrientation, d
 				}
 
 				FlyoutPage.UpdateFlyoutLayoutBehavior();
-				MessagingCenter.Send<IVisualElementRenderer>(this, NavigationRenderer.UpdateToolbarButtons);
+
+				WeakReferenceMessenger.Default.Send(new UpdateToolBarButtonsMessage());
```

Comment:
Nit: this can be simplified:

```suggestion
				WeakReferenceMessenger.Default.Send<UpdateToolBarButtonsMessage>();
```

Same for the other two occurrences below in this file.